package.json

```
@@ -210,7 +206,7 @@
     "electron-devtools-installer": "^3.2.0",
     "electron-download": "^4.1.1",
     "electron-mock-ipc": "^0.3.12",
-    "electron-packager": "^14.2.1",
+    "electron-packager": "15.3.0",
```

```suggestion
    "electron-packager": "^15.3.0",
```

This allows us to update to a newer version when there are no breaking changes, per the [Semantic Versioning](https://semver.org/).